test/Microsoft.AspNetCore.Mvc.IntegrationTests/ActionParametersIntegrationTest.cs

```diff
@@ -383,6 +383,68 @@ private class Person5
             Assert.Empty(modelState.Keys);
         }
 
+        [Theory]
+        [InlineData(typeof(ClassWithNoDefaultConstructor), "Name")]
+        [InlineData(typeof(StructWithNoDefaultConstructor), "Name")]
+        [InlineData(typeof(AbstractClassWithNoDefaultConstructor), "Name")]
+        public async Task ActionParameter_NoDefaultConstructor_Fails(Type parameterType, string key)
+        {
+            // Arrange
+            var argumentBinder = ModelBindingTestHelper.GetArgumentBinder();
+            var parameter = new ParameterDescriptor()
+            {
+                ParameterType = parameterType
+            };
+            var testContext = ModelBindingTestHelper.GetTestContext(request =>
+            {
+                request.QueryString = QueryString.Create(key, "James");
+            });
+
+            // Act & Assert
+            var exception = await Assert.ThrowsAsync<InvalidOperationException>(async () =>
+            {
+                await argumentBinder.BindModelAsync(parameter, testContext);
+            });
```

Comment:
Get rid of the ceremonial braces and `async` / `await`: `await Assert.ThrowsAsync<...>(() => argumentBinder.BindModelAsync(parameter, testContext));`